packages/assets-controllers/src/assetsUtil.ts

```diff
@@ -16,6 +17,14 @@ import type {
   OpenSeaV2Nft,
 } from './NftController';
 import type { ApiNft, ApiNftContract } from './NftDetectionController';
+import type { AbstractTokenPricesService } from './token-prices-service';
+import { type ContractExchangeRates } from './TokenRatesController';
+
+/**
+ * The maximum number of token addresses that should be sent to the Price API in
+ * a single request.
+ */
+export const TOKEN_PRICES_BATCH_SIZE = 100;
```

Comment:
Should this be a package-level export as well? I assume it's only used inside of `assets-controllers`, but just wanted to check.

Reply:
I had requested that it be moved from TokenRatesController so it wasn't a package-level export, as I didn't think anyone needed to refer to this. But I guess I made an assumption, so that's a fair question — do we anticipate this being used in client code?

Reply:
For now i do not expect extension to be using this.
You guys think its fine to keep it in assetsUtil.ts in that case?

Reply:
I don't anticipate that we'd need this anywhere else either.

It seems more appropriate for this to be part of the service itself, rather than here or in the controller. This choice in batch size is a recommendation specific to the CodeFi V2 price API in particular. It would be ideal if we could attach this batch size to the service instead.

Reply:
This constant exists here because the batching code is outside of the service at the moment. Perhaps we can integrate that better in another PR?